Library/ViewModels/PledgeViewModel.swift

```diff
@@ -1194,3 +1184,23 @@ private func pledgeAmountSummaryViewData(
     shippingAmountHidden: !shippingViewsHidden
   )
 }
+
+private func checkoutPropertiesData(
```

Comment:
Let's delete this helper method. I feel like we have something similar in a few different files and it's not very intuitive.

Reply:
I agree with you, that's why I said in my previous comment that it's not necessary, I thought we can give it a try following your comment.